frontend/app/routes/_public+/apply+/$id+/review-information.tsx

```diff
@@ -254,15 +260,18 @@ export default function ReviewInformation() {
         </DescriptionListItem>
       </dl>
       <h2 className="mb-5 mt-8 text-2xl font-semibold">{t('apply:review-information.submit-app-title')}</h2>
-      <p className="mb-4">{t('apply:review-information.submit-p-one')}</p>
-      <p className="mb-4">{t('apply:review-information.submit-p-two')}</p>
-      <h3 className="font-semibold">{t('apply:review-information.print-app-title')}</h3>
-      <p className="mb-4">{t('apply:review-information.submit-p-three')}</p>
-      <Form method="post">
-        <Button id="confirm-button" variant="green">
-          {t('apply:review-information.submit-button')}
-        </Button>
-      </Form>
+      <p className="mb-4">{t('apply:review-information.submit-p-proceed')}</p>
+      <p className="mb-4">{t('apply:review-information.submit-p-false-info')}</p>
+      <div className="flex">
+        <ButtonLink to={`/apply/${id}/exit-application`} className="mr-4 rounded  border border-sky-900 bg-white">
+          <span className="font-semibold">{t('apply:review-information.exit-button')}</span> <FontAwesomeIcon icon={faX} className="h-3 w-3 pl-2" />
+        </ButtonLink>
+        <Form method="post">
+          <Button id="confirm-button" variant="green" className="font-semibold">
+            {t('apply:review-information.submit-button')}
+          </Button>
+        </Form>
+      </div>
```

Comment:
Just so we're consistent with the rest of the buttons in this apply flow:

```suggestion
      <div className="flex flex-wrap items-center gap-3">
        <ButtonLink to={`/apply/${id}/exit-application`} variant="alternative">
          <span className="font-semibold">{t('apply:review-information.exit-button')}</span> <FontAwesomeIcon icon={faX} className="pl-2" />
        </ButtonLink>
        <Form method="post">
          <Button id="confirm-button" variant="green">
            {t('apply:review-information.submit-button')}
          </Button>
        </Form>
      </div>
```